charts/mlp/templates/postgres-service.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if not .Values.postgresql.enabled }}
```

Comment:
Why are we adding this service rather than directly use the provided external address in the template for `postgres.host`, like we handle the username and password?

Reply:
You could, but I find it neater to put it as a service because the same address can be used regardless of PostgreSQL using helm charts vs the external versions.